src/aas-api-webapp-aasxfile/Startup.cs

```diff
@@ -47,15 +48,20 @@ public void ConfigureServices(IServiceCollection services)
             services
                 .AddMvc(options =>
                 {
-                    options.InputFormatters.RemoveType<Microsoft.AspNetCore.Mvc.Formatters.SystemTextJsonInputFormatter>();
-                    options.OutputFormatters.RemoveType<Microsoft.AspNetCore.Mvc.Formatters.SystemTextJsonOutputFormatter>();
+                    options.InputFormatters
```

Comment:
Why are changing the code of the AASX File server? Your changes are for the Repository! Did you test this change?